application/templates/static_site/measure.html

```diff
@@ -527,19 +530,23 @@ <h2 id="download-the-data" class="heading-medium">Download the data</h2>
                           version=measure_version.version,
                           filename=file.file_name)
                 %}
-                <a href="{{ measure_version_data_url }}" data-on="click" data-event-category="CSV downloaded" data-event-action="Source data" data-event-label="{{ file.title }}">
-                      {{ file.title }}
-                      {{ " - Spreadsheet " if file.extension() == "csv" }}
-                      {{ " - Microsoft Excel " if file.extension() == "xls" or file.extension() == "xlsx" }}
-                      {{ " - Open Document Format " if file.extension() == "odf" }}
-                      ({{ file.extension() }})
-                      {{ file.size|filesize }}
-                </a>
-                {% if file.description %}
-                  <p>{{ file.description }}</p>
-                {% else %}
-                  <br />
-                {% endif %}
+
+                <div itemprop="distribution" itemscope itemtype="http://schema.org/DataDownload">
+                  <meta itemprop="encodingFormat" content="text/csv">
```

Comment:
Should the content type be derived a little more intelligently? It looks like we support data in csv/xls(x)/odf, so this probably shouldn't be a static string for the csv type?

Reply:
I think we once supported alternative formats in theory, but now we only support CSV? 🤷‍♂️

Reply:
Yep, looks like you're right 👌